<?xml version="1.0"?>
 
<launch>
    
   <node name="pc_pipeline" pkg="pc_pipeline" type="pc_pipeline_node" output="screen" >
   <param name="scan_name" type="string" value="/rslidar_points" />
   <param name="scan_queue_size" type="int" value="2" />
   <param name="scan_modified_name" type="string" value="/filtered_cloud" />
   <param name="scan_modified_queue_size" type="int" value="2" />

   <param name="pipeline_mode" type="int" value="2" />
   <param name="segmentation" type="bool" value="true" />
   </node>

   <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_pipeline)/rviz/lidar.rviz"/>
   
   <node pkg="rosbag" type="play" name="player" output="screen" args="-l /home/arthur/dataset1/data1.bag --pause -q -s 21"/>   

</launch>

